libs/auth/src/artifact_authentication.ts

```diff
@@ -54,12 +93,13 @@ interface ArtifactSignatureBundle {
  *
  * File paths are sorted alphabetically to ensure a consistent output.
  */
-async function hashDirectoryContents(directory: string): Promise<Buffer> {
-  const filePaths = (await recursiveReadDir(directory)).sort();
+/* istanbul ignore next */
```

Comment:
so `hashDirectoryContents` was only for legacy CVRs - for new CVRs we don't have to hash directory contents because we are only hashing the tree's top-level file...

Reply:
Yeah that's essentially right! Though just to be extra precise, we're hashing and signing the tree's top-level file, but that top-level file further contains a hash of everything else in the directory. So we're still indirectly incorporating everything into the signature file (which is a must)